eng/Version.Details.xml

```diff
@@ -115,6 +115,10 @@
       <Sha>b4fa7f2e1e65ef49881be2ab2df27624280a8c55</Sha>
       <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.WorkloadTesting.Internal" Version="9.0.0-preview.3.24151.4">
```

Comment:
NIT: Can we move this new dependency up to the section where we have the rest of the dotnet-runtime repo dependencies?